### PR TITLE
[DOCFIX] Update Cluster doc

### PR DESCRIPTION
**Pull request by HelloHorizon**

### CODE REVIEW

File: docs/en/deploy/Running-Alluxio-On-a-Cluster.md [hunk 1]
```diff
@@ -217,6 +217,6 @@ and then [restart the service](#restart-alluxio).
 ### Update Worker-side Configuration
 
 If you only need to update some local configuration for a worker (e.g., change the mount
```
Comment (bf8086) [inline docs/en/deploy/Running-Alluxio-On-a-Cluster.md:219]: Would it be "change the amount of storage" instead?

Comment (HelloHorizon) [reply docs/en/deploy/Running-Alluxio-On-a-Cluster.md:219]: You are right, I do not really understand this sentence at the beginning.

**Comment by bf8086** [comment]
alluxio-bot, merge this please.

**Comment by bf8086** [comment]
alluxio-bot, cherry-pick this to branch-2.2 please.

**Comment by bf8086** [comment]
alluxio-bot, cherry-pick this to branch-2.3 please.

**Pull request merged**